packages/default-storage/example/examples/tests.ts

```diff
@@ -42,10 +41,38 @@ const tests: Record<string, TestStep[]> = {
         name: "Sarah",
         age: "23",
         eyesColor: "green",
+      },
+      expected: {
+        name: "Sarah",
+        age: "23",
+        eyesColor: "green",
         shoeSize: "10",
       },
     },
   ],
+  "Should merge items #2": [
```

Comment:
```suggestion
  "Should keep existing entries when merging with an empty object": [
```

Reply:
ah, hate that long name in test case
`Should keep existing entries when merging with an empty object with delegate` 😄 

Let's have it in tho